std/experimental/allocator/building_blocks/kernighan_ritchie.d

```diff
@@ -8,7 +8,7 @@ version(unittest) import std.conv : text;
 // KRRegion
 /**
 $(D KRRegion) draws inspiration from the $(LINK2
-std_experimental_allocator_region.html, region allocation strategy) and also the
+std_experimental_allocator_building_blocks_region.html, region allocation strategy) and also the
```

Comment:
`LINK2` shouldn't be used for phobos references. It doesn't work on the /library/ pages.

Use `MREF` instead: `$(MREF std,experimental,allocator,building,blocks,region)`